### DISCUSSION TITLE
Clarity datepicker component - Ability to disable particular dates in the date picker

### DISCUSSION THREAD

**Discussion by gayathrirajendran** [vmware/clarity#6157] [Q&A] [answered]
Can I find information on when this feature request will be available ? There are a number of use cases for this, like disabling holidays (like saturday, sunday, diwali, christmas...)
@gnomeontherun 
https://github.com/vmware/clarity/issues/3009

**Comment by gnomeontherun** [comment, marked as answer]
We are no longer planning to add features to Clarity Angular, which is where the Datepicker lives. When we rebuild it with Clarity Core, we will be considering adding capabilities like this. Right now the picker can disable with min or max dates.
